styles.css

```diff
@@ -1,65 +1,32 @@
-/* ******GLOBAL******** */
+/* ****** GENERAL STYLES ******** */
```

Comment:
As a general note: I would recommend separating each style block with a space, just for better dev readability!